### DISCUSSION TITLE
Can see the car movement track/car position information in real time?

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#122] [General]
Is there a configuration that allows us to see the process of the vehicle moving, rather than going directly from one point to another, if you have some good idea to this problem, please tell me in time, thanks very much.

**Comment by swltr** [comment]
@HuJunnj Yes, that is possible - if the vehicle driver provides the vehicle's precise position in addition to its logical one.

**Reply by HuJunnj** [reply]
thanks very much for you replay me that this problem.  I can help complete the international language translation of Chinese in the system if you need me to help.
This means that you must implement this feature yourself, The system does not provide the existing setting properties to obtain the precise location of the vehicle?

BR
Jun

**Reply by swltr** [reply]
Whether a given vehicle (driver) provides detailed information about the vehicle's current position or just the logical position is something that is specific to the vehicle type / communication interface. The openTCS API provides a method `VehicleProcessModel.setVehiclePrecisePosition()` with which this information can be set on the vehicle element in the plant model. Whether it is called or not is up to the vehicle driver.

**Comment by HuJunnj** [comment]
okay l see, thank you very much



---- Replied Message ----
| From | Stefan ***@***.***> |
| Date | 01/30/2024 03:45 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [openTCS/opentcs] Can see the car movement track/car position information in real time? (Discussion #122) |

@HuJunnj Yes, that is possible - if the vehicle driver provides the vehicle's precise position in addition to its logical one.
